modules/sdk/src/main/scala/org/tessellation/sdk/cli/StateChannelWhitelist.scala

```diff
@@ -0,0 +1,31 @@
+package org.tessellation.sdk.cli
+
+import cats.data.NonEmptySet
+
+import scala.io.Source
+
+import org.tessellation.schema.address.Address
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.sdk.config.AppEnvironment
+import org.tessellation.sdk.config.AppEnvironment._
+
+import io.circe.parser.decode
+
+object StateChannelWhitelist {
+
+  def get(env: AppEnvironment): Option[Map[Address, NonEmptySet[PeerId]]] =
+    env match {
+      case Dev     => None
+      case Testnet => Some(loadTestnet())
+      case Mainnet => Some(loadMainnet())
+    }
+
+  def loadTestnet(): Map[Address, NonEmptySet[PeerId]] = load("state_channels_whitelists_testnet.txt")
+  def loadMainnet(): Map[Address, NonEmptySet[PeerId]] = load("state_channels_whitelists_mainnet.txt")
+  def load(name: String): Map[Address, NonEmptySet[PeerId]] = Source
+    .fromResource(name)
+    .getLines()
+    .map(decode[(Address, NonEmptySet[PeerId])])
+    .map(_.fold(e => throw e, identity))
```

Comment:
is it safe to throw like that?

Reply:
It will be thrown during sartup if cannot parse file placed in resources. I think it is desired behaviour, we don't want to start if this list is somehow invalid. 
Nevertheless, I have provided tests, which check these files, so it shouldn't happen.